easybuild/easyconfigs/n/nbclassic/nbclassic-1.3.1-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'PythonPackage'
+
+name = 'nbclassic'
+version = "1.3.1"
+
+homepage = 'https://jupyter.org/'
+description = """NbClassic provides a backwards compatible Jupyter Notebook interface
+that you can install side-by-side with the latest versions: That way, you can fearlessly
+upgrade without worrying about your classic extensions and customizations breaking."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('maturin', '1.6.0'),
```

Comment:
If it needs maturin it should also need Rust.
and if it needs Rust, it should be a CargoPythonPackage instead, and should include list of crates for caching.

But I can't find that this actually needs maturin or Rust at all looking at the source code?

Reply:
`maturin` can indeed be removed, I just tested it and `nbclassic` built just fine